models.py

```diff
@@ -342,3 +376,5 @@ def save(self, force_insert=False, force_update=False, using=None,
 
 
 # needed to load signals
+# noinspection PyUnresolvedReferences
```

Comment:
Accidentally deleted these. :(

Reply:
I mean, the statement below this.
This line is to prevent PyCharm from deleting the line when optimizing the import